db/views/hmis_group_viewable_entity_projects_v02.sql

```diff
@@ -0,0 +1,50 @@
+-- rails generate scenic:view hmis_group_viewable_entity_projects --replace
+(
+  -- projects
+  SELECT
+    "hmis_group_viewable_entities"."id" AS group_viewable_entity_id,
```

Comment:
The only change in this view is renaming `group_viewable_entities`=>`hmis_group_viewable_entities`